packages/fxa-settings/src/pages/Signin/InlineRecoverySetup/index.tsx

```diff
@@ -0,0 +1,225 @@
+                  </p>
+                </FtlMsg>
+                <div className="input-row">
+                  <InputText
+                    label={localizedInputTextLabel}
+                    className="tooltip-below recovery-code text-start m-4"
+                    anchorStart={true}
+                    required
+                    autoFocus
+                    autoComplete="off"
+                  />
+                </div>
+                <FtlMsg id="confirm-button">
+                  <button type="submit" className="cta-primary px-10 py-2 mt-2">
+                    Confirm
+                  </button>
+                </FtlMsg>
+                <div className="flex justify-between mx-4 mt-4">
+                  <FtlMsg id="back-link">
+                    <button className="link-blue text-sm">Back</button>
+                  </FtlMsg>
+                  <FtlMsg id="cancel-setup">
+                    <button className="link-blue text-sm">Cancel setup</button>
+                  </FtlMsg>
+                </div>
+              </div>
+            </form>
+          </section>
+        </>
+      ) : (
+        <>
+          <div className="mb-4">
+            {serviceName ? (
+              <FtlMsg id="inline-recovery-setup-header" vars={{ serviceName }}>
+                <h1 className="card-header">
+                  Save backup authentication codes{' '}
+                  <span className="card-subheader">{`to continue to ${serviceName}`}</span>
+                </h1>
+              </FtlMsg>
+            ) : (
+              <FtlMsg id="inline-recovery-setup-header-default">
+                <h1 className="card-header">
+                  Save backup authentication codes{' '}
+                  <span className="card-subheader">
+                    to continue to account settings
+                  </span>
+                </h1>
+              </FtlMsg>
+            )}
+          </div>
+          <section>
+            <FtlMsg id="inline-recovery-setup-message">
+              <p className="text-sm mb-6">
+                Store these one-time use codes in a safe place for when you
+                don’t have your mobile device.
+              </p>
+            </FtlMsg>
+            <div>
+              <div className="flex rounded-xl px-7 font-mono text-center text-sm text-green-900 bg-green-800/10 flex-wrap relative mb-6 py-4">
+                {recoveryCodes.map((code) => {
+                  return <div className="flex-50% py-1">{code}</div>;
+                })}
+              </div>
+              {isIos ? (
+                <div className="flex justify-center w-4/5 max-w-48 mx-auto fill-grey-500">
+                  <button
+                    title={localizedCopyButtonTitle}
+                    className="w-16 h-16 relative inline-block text-grey-500 rounded active:text-blue-600  active:fill-blue-600 focus:outline focus:outline-2 focus:outline-offset-2 focus:outline-blue-500 hover:bg-grey-50"
```

Comment:
FWIW, other buttons seem to be `w-12 h-12`, but this is `w-16 h-16`.

Reply:
Thanks for catching that!!